packages/cli/generators/repository/templates/src/repositories/repository-crud-default-template.ts.ejs

```diff
@@ -10,6 +10,6 @@ export class <%= className %>Repository extends <%= repositoryTypeClass %><
   constructor(
     @inject('datasources.<%= dataSourceName %>') dataSource: <%= dataSourceClassName %>,
```

Comment:
Please note that we should be using `dataSource` (not `datasource`) in all places since my recent commit https://github.com/strongloop/loopback-next/commit/69506a44cf25d3c8d0b50d0758e8f5b808852917